### PR TITLE
Added Shield Shield pull-up resistor notes

### PR DESCRIPTION
**Pull request by harrisonhjones**
Adds pull-up resistor notes to Shield Shield in both Core and Photon sections.

### CODE REVIEW

File: src/content/datasheets/photon-shields.md [hunk 1]
```diff
@@ -23,14 +23,17 @@ We use Texas Instruments TXB0108PWR to do the voltage translation in between the
 
 The new version of the Shield Shield (v3.x.x) uses dedicated mosfet based voltage translation on the I2C lines. We also decided to add a prototyping area in empty space in the middle of the shield.
 
+#### Notes
+
+1. One drawback of using the TXB0108PWR as a voltage translator is that it is only capable of driving loads at short distances. Long length wires will introduce excessive capacitive loading and cause the auto direction detection to fail. To overcome this drawback, the shield shield also has an optional on-board 74ABT125 buffer that is capable of driving heavier loads in *one* direction. A user can jumper wire to whichever IO pin they would like to be translated to 5V.
```
Comment (kennethlimcp) [inline src/content/datasheets/photon-shields.md:28]: Hmm isn't the new shield using another IC?

Comment (harrisonhjones) [inline src/content/datasheets/photon-shields.md:28]: Not according to the [datasheets](https://github.com/spark/shields/tree/master/photon-shields/shield-shield/datasheets)

File: src/content/datasheets/core-shields.md [hunk 2]
```diff
@@ -21,6 +21,11 @@ Due to the limited number of pin to function combinations, we have only mapped t
 
 The shield has an onboard voltage regulator and can be powered from 7V to 15V DC. You could also power it via the USB plug on the Spark Core alone but the current would be limited to 500mA.
 
+#### Notes
```
Comment (kennethlimcp) [inline src/content/datasheets/core-shields.md:24]: This is referencing to the old shield shield so the notes here does not count.

Comment (kennethlimcp) [inline src/content/datasheets/core-shields.md:24]: @harrisonhjones, can you fix this or i will do it manually.

**Comment by monkbroc** [comment]
The time has passed for the shield shield.

**Pull request closed without merging**